features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -314,6 +314,9 @@
   "authentication.jit_provisioning.username_provisioning_url": "/accountrecoveryendpoint/register.do",
   "authentication.jit_provisioning.password_provisioning_url": "/accountrecoveryendpoint/signup.do",
 
+  "LegacyFeatures.EnableOAuth1": false,
+  "LegacyFeatures.EnableIdentityConnectDCR": false,
```

Comment:
use lower case to maintain the consistency with other configs defined in default.json

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2997/commits/2ce0e366e148c159060aa4ad2103ae94416682bb